### ISSUE TITLE
[Android][NOTE] Google will deprecate expansion files (obb)! Play Asset Delivery will be needed

### ISSUE THREAD

**Issue by Oldes** [Gamua/Adobe-Runtime-Support#439]
## Problem Description

As I'm forced to update our Google's expansion files downloader ANE (again) to be working with API29, I noticed [in this page](https://developer.android.com/google/play/expansion-files):

> Important: In the second half of 2021, new apps will be required to publish with the Android App Bundle on Google Play. New apps larger than 150 MB must use either [Play Feature Delivery](https://developer.android.com/guide/app-bundle/dynamic-delivery) or [Play Asset Delivery](https://developer.android.com/guide/app-bundle/asset-delivery).

So I hope that **Android App Bundle** packing will be working when it comes.. and I wonder if the assets delivery will have to be resolved by using custom ANEs or also somehow **supported by the AIR packing process**.

**Comment by Oldes** [comment]
@ajwfrost can I kindly remind you this Google's upcoming requirement and ask you if Harman is aware of it?  Thank you.

Is there any chance that Google will help you with it as they did with Unity and Unreal?

**Comment by ajwfrost** [comment]
@Oldes yes we're aware of this one and are working on the App Bundle stuff again. We have some contacts within Google on this, will have to see whether we need their help or not - I think from one of the updates I saw, we may be okay! but will bear that in mind :-)
thanks

**Comment by PippoApps** [comment]
Any news on this?
I can't find a way on Play Console to upload OBB files (the new console UX seems to be designed by Kafka and Hoffman together).

**Comment by ajwfrost** [comment]
Hi @PippoApps  -> you can hopefully use the Play Asset Delivery ANE for this sort of thing now:
https://github.com/airsdk/ANE-PlayAssetDelivery

**Comment by PippoApps** [comment]
Gotcha.
Thanks!
